test-runner/src/Editor.js

```diff
@@ -40,7 +40,7 @@ import {
 const completePropertyAfter = ["PropertyName", ".", "?."];
 
 const fixedHeightEditor = EditorView.theme({
-  "&": { height: "300px" },
+  "&": { height: "calc(50vh - 100px)" },
```

Comment:
I don't love this. I wanted to try and make these code windows dynamically sized with flexbox. However, I couldn't make them take up the maximum area and then scroll. I think it might be because there is relative and absolute positioning done in the component. This feels like an okay hack for now.